website/pages/chapters/assessing_performance.js

```diff
@@ -193,22 +237,37 @@ export default function AssessingPerformance() {
           dealing with the <IM math={`(x, y)`} /> pairs. So then selecting the
           model that generalizes best, would then mean choosing the one with the
           lowest true error.
+          <MatthewComment>
+            I don't understand how these last two sentences are related to the
```

Comment:
I was getting more at the fact that it doesn't look like an expectation they might have seen, but rather an expectation on a joint distribution. Will clarify.